demisto_sdk/commands/secrets/secrets.py

```diff
@@ -222,7 +224,7 @@ def remove_whitelisted_items_from_file(file_content: str, secrets_white_list: li
             str: The file content with the whitelisted items removed.
         """
         for item in secrets_white_list:
-            file_content = file_content.replace(item, '')
+            file_content = re.sub(WHILEIST_REGEX.format(item), '', file_content)
```

Comment:
Why do you prefer to use regex ?
It seems that in this case we just need to replace string as it is

Reply:
Because we use the secrets also as substring so "anyrun.com" secret will also replace "https://anyrun.com/api/v2".
Else, it'll replace and will find "https:///api/v2" as a secret as well.

Reply:
ok